core/redfishCommand.py

```diff
@@ -95,13 +81,27 @@ def execute(cls, redfishConfig, command, echo = False):
                     handlerFile = 'commands/' + brand + '/' + words[0] + '_' + words[1] + '.py'
                     handlerName = 'commands.' + brand + '.' + words[0] + '_' + words[1]
 
-            # Check common version of command
+            # Check brand version of command
             if (path.exists(handlerFile) == False):
                 Trace.log(TraceLevel.TRACE, '++ file ({}) does not exist, check common folder'.format(handlerFile))
-                Trace.log(TraceLevel.ERROR, 'Command file ({}) does not exist!'.format(handlerFile))
-                return None
+                brand = 'common'
+                if (len(words) == 1):
+                    handlerFile = 'commands/' + brand + '/' + words[0] + '.py'
+                    handlerName = 'commands.' + brand + '.' + words[0]
+                elif (len(words) >= 2):
+                    if (words[0] == 'help' or words[0] == 'assert' or words[0] == 'version'):
+                        handlerFile = 'commands/' + brand + '/' + words[0] + '.py'
+                        handlerName = 'commands.' + brand + '.' + words[0]
+                    else:
+                        handlerFile = 'commands/' + brand + '/' + words[0] + '_' + words[1] + '.py'
+                        handlerName = 'commands.' + brand + '.' + words[0] + '_' + words[1]
```

Comment:
While this is a minimal optimization, I'd suggest the use of python 3's `f-string` here.
They offer slightly faster execution and a (subjectively) cleaner format. (stackoverflow [thread](https://stackoverflow.com/questions/38722105/format-strings-vs-concatenation))
`handlerName = f'commands.{brand}.{words[0]}_{words[1]}'`

Again, what you have here is fine.  This is just a suggestion.  Of all the different ways to format strings in Python, I've found `f-string`'s to be the easiest to work with.